dynawo/cmake/SetPatchCommands.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+function( SetPatchCommands projectName)
+
+    set( TmpDirName "$ENV{DYNAWO_HOME}/dynawo/3rdParty")
```

Comment:
tab is two spaces wide (and not real tab either)

Reply:
OK, 'expand --tab=2' will do the job. I used --tab=4 until now.